### PR TITLE
docs: include README section on locking stuff

### PR DESCRIPTION
**Pull request by EthanFreestone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,37 @@ Developers looking to access the services exposed by mod-agreements can find mor
...
+In order of importance to check:
+
+- **Liquibase**
+  - The module uses liquibase in order to facilitate module data migrations
+  - Unfortunately this has a weakness to shutdown mid migration.
+  - Check `<tenantName>_mod_agreements.tenant_changelog_lock` does not have `locked` set to `true`
+    - If it does, the migration (and hence the upgrade itself) have failed, and it is difficult to extricate the module from this scenario.
+    - It may be most prudent to revert the data and retry the upgrade.
+  - In general, while the module is uploading it is most likely to succeed if after startup and tenant enabling/upgrading through okapi that the module and its container are NOT KILLED for at least 2 minutes.
+  - An addition, a death to the module while upgrading could be due to a lack of reasonable resourcing making it to the module
```
Comment (ostephens) [inline README.md:37]: Is it worth mentioning CPU resourcing explicitly here as something to check?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,39 @@ Developers looking to access the services exposed by mod-agreements can find mor
 The module has important dependences on reference data. initial installations and module upgrades should specify loadReference=true. The module
 may not work as expected if this is omitted.
 
+While this README and the module description template offer some guidance on how to run this module and the resourcing required, it is not possible to anticipate all possible environmental configurations and deployment processes. Determining the exact resourcing, deployment processes and other aspects such as the size of database conneciton pools needed in any particular environment is down to those running the moule and it is recommended that all such practices are fully documented by those responsible
```
Comment (ostephens) [inline README.md:23]: Typo: moule instead of module

Comment (EthanFreestone) [reply README.md:23]: :pray:

File: README.md (hunk 1 above)
Comment (EthanFreestone) [reply README.md:39]: Have now done, along with a link to module requirements page for Ramsons (as an example)

**Review by ostephens — APPROVED** [review]
LGTM

**Pull request merged**